integration_test.go

```diff
@@ -89,13 +89,10 @@ func verifyBaggage(ctx context.Context) error {
 		return errors.New("missing span")
 	}
 
-	spanCtx, ok := span.Context().(jaeger.SpanContext)
+	_, ok := span.Context().(jaeger.SpanContext)
 	if !ok {
```

Comment:
nit: can you inline the `ok := ` into the if,
```go
if _, ok := span.Context().(jaeger.SpanContext); !ok {
  [...]
}
```

Reply:
Ya sure.